ex2_hal/dfgm/equipment_handler/include/dfgm_handler.h

```diff
@@ -76,6 +77,7 @@ typedef enum {
 #define DFGM_QUEUE_DEPTH 1248
 #define DFGM_FILE_NAME_MAX_SIZE 25
 #define DFGM_RX_TASK_SIZE 500
+#define DFGM_HK_COLLECTION_MAX_RUNTIME 3000 // in ticks, assuming 1000 ticks/sec
```

Comment:
I think it would be better to use 3*ONE_SECOND that we define in system.h!

Reply:
Alright sounds good! Will do that now